sdk/webpubsub/Azure.Messaging.WebPubSub/README.md

```diff
@@ -36,11 +37,12 @@ dotnet add package Azure.Messaging.WebPubSub
 - An [Azure subscription][azure_sub].
 - An existing Azure Web PubSub service instance.
 
-### Create and authenticate a `WebPubSubServiceClient`
+### Authenticate the client
```

Comment:
From Web PubSub service's perspective of view, `client` has its special meaning, so we'd like to keep the original **Create and authenticate a `WebPubSubServiceClient`** to avoid confusion to customers

Reply:
Added back the exception for WebPubSub.